.github/workflows/vib.yaml

```diff
@@ -0,0 +1,45 @@
+name: 'vib'
+on: # rebuild any PRs and main branch changes
+  pull_request:
+    branches:
+      - stable
+    types:
+      - assigned
```

Comment:
Why would you want to re-run the workflow if the PR is assigned? That even't doesn't mean code changes, does it?

Reply:
I completely agree, I will remove it.